File: Programming/Arrayprogrammings/SortingofArray.js
var arr =[0,-1,5,6,1,2,3]
for (let index = 0; index < arr.length; index++) {
    for (let j= 0; j< arr.length; j++) {
        if(arr[j]>arr[j+1])
        {
            var temp=arr[j]
            arr[j]=arr[j+1]
            arr[j+1]=temp
        }   
    }
}
console.log(arr)

File: javaPrograming/SDETProgreamming/src/arraysprogramming/SortingOfArray.java
package arraysprogramming;

public class SortingOfArray {

	public static void main(String[] args) {
		int arr[]= {1,2,5,4,1,4,4};
		for (int i = 0; i < arr.length; i++) {
			for (int j = 1; j < arr.length-i; j++) {
				if(arr[j-1]>arr[j]) {
					int temp=arr[j-1];
					arr[j-1]=arr[j];
					arr[j]=temp;	
				}	
			}
		}
		for(int a:arr) {
			System.out.println(a);
		}
	}
}
